code/game/machinery/computer/camera_advanced.dm

```diff
@@ -171,9 +174,9 @@
 			C.images -= user_image
 
 /mob/camera/aiEye/remote/Destroy()
-	eye_user = null
 	origin = null
-	return ..()
+	. = ..()
+	eye_user = null
```

Comment:
Care to explain this one?

Reply:
As the commit message states, this fixes remote camera eyes not automatically removing static images in Destroy(), as pointed out by @AnturK in #38209.